integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseCliOpts.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * An object that holds all the options from the command line
+ * when Alluxio fuse is being launched through CLI.
+ */
+public final class AlluxioFuseCliOpts {
+  private final String mMountPoint;
+  private final String mAlluxioPath;
+  private final List<String> mFuseOptions;
+
+  private AlluxioFuseCliOpts(@Nullable String mountPoint, @Nullable String alluxioPath,
+      @Nullable List<String> fuseOptions) {
+    mMountPoint = mountPoint;
+    mAlluxioPath = alluxioPath;
+    mFuseOptions = fuseOptions;
+  }
+
+  /**
+   * Gives the command line arguments to parser.
```

Comment:
Description in user level understanding instead of code level.
e.g.

Contructs a {@link AlluxioFuseCliOpts} based on user command line input.